### ISSUE TITLE
[UI] Simplify rebinding all keys in a particular category

### ISSUE THREAD

**Issue by rswgnu** [saulpw/visidata#743]
Think about finding a library that lets you allow for more flexible key sequences like Emacs where related bindings are all grouped under a prefix key, often a control key, and are easy to examine within that single key map.  Then one could explore sheet keys, filter keys, selection keys, all as a group by the type of operation or entity.

This would also help make VisiData more accessible to people who have models of how they want their key sequences to work.

**Comment by saulpw** [comment]
Hi @rswgnu , thanks for the suggestion.  This would be a big project as stated.  I think your underlying desire could be fulfilled more reasonably by some other means, either already existing or more in line with the current roadmap.

That said, you're welcome to try and hack around with `vd.allPrefixes` and some kind of mass binding scheme.  See visidata.org/docs/api/commands.  Let me know if you do come up with something more like Emacs, and we can share it as a plugin or something.

**Comment by rswgnu** [comment]
I will have a look and feedback if I get anywhere.

**Comment by rswgnu** [comment]
I have made some progress with Emacs bindings and will share when in a reasonable state.  However, only the single keys are working and when I try to set a new prefix key in visidata that is already bound as a single key, it fails to override as a prefix.

Here is a sample of my .visidatarc file that does not work.  Question is what else has to be done to create a prefix key (e.g. unbind it first) and how does one just unbind a key?

#######

vd.allPrefixes = ['^H', '^X', '^C', 'g', 'z', '^[']
bindkey = visidata.vd.bindkey
bindkey('^H^H', 'sysopen-help')
bindkey('^Hl', 'cmdlog-all')
bindkey('^X=', 'describe-selected')
bindkey('^X-', 'describe-sheet')
bindkey('^X+', 'describe-all')
bindkey('^Xo', 'splitwin-swap') # Jump to other sheet
bindkey('^X^S', 'save-sheet')
bindkey('^X^F', 'open-file')
bindkey('^X^H', 'freq-col') # Frequency of values in columns
bindkey('^X^S', 'save-sheet')

**Comment by rswgnu** [comment]
I figured it out for all prefixes I want right now other than ^H which seems to be hardcoded to help display.

I needed to add:

unbindkey = BaseSheet.unbindkey
[unbindkey(keys) for keys in ('^H', '^X', '^C')]
visidata.vd.allPrefixes = ['^X', '^H', '^C', 'g', 'z', '^[']

Also, it would be great if for control-left-arrow one could use the name '^KEY_LEFT' instead of 'kLFT5' which is showing as the name under MacOS.

But I'm getting there.  I just want this stuff to be easier for others to modify down the road.
